isis/tests/CMakeLists.txt

```diff
@@ -43,6 +47,6 @@ add_executable(runISISTests
                IsisTestMain.cpp
                ${test_source})
 
-target_link_libraries(runISISTests isis ${MISSION_LIBS} ${ALLLIBS} ${GTEST_LIBRARIES} ${GTEST_MAIN_LIBRARIES} Threads::Threads)
+target_link_libraries(runISISTests isis ${MISSION_LIBS} ${ALLLIBS} gmock_main)
```

Comment:
Do you know why threads are no longer needed?

Reply:
Because the gmock target already includes it as its own dependency. Another nice bit of target based linking is that it handles dependencies of dependencies much better.